api/src/identity-access-management/domain/services/refresh-token-service.js

```diff
@@ -56,10 +57,14 @@ async function createRefreshTokenFromUserId({ userId, source, uuidGenerator = ra
  * @typedef {function} createAccessTokenFromRefreshToken
  * @param {Object} params
  * @param {string} params.refreshToken
+ * @param {string} params.scope
  * @return {Promise<{expirationDelaySeconds: number, accessToken: string}>}
  */
-async function createAccessTokenFromRefreshToken({ refreshToken }) {
-  const { userId, source } = (await findByRefreshToken(refreshToken)) || {};
+async function createAccessTokenFromRefreshToken({ refreshToken, scope: targetScope }) {
+  const { userId, source, scope } = (await findByRefreshToken(refreshToken)) || {};
```

Comment:
Mettre à jour la jsdoc du findByRefreshToken un peu plus haut pour ajouter le scope là : https://github.com/1024pix/pix/blob/pix-13911-apply-scope-in-refresh-token/api/src/identity-access-management/domain/services/refresh-token-service.js#L15